### ISSUE TITLE
questions about atst-frame evaluated on audioset_strong_eval

### ISSUE THREAD

**Issue by Ming-er** [Audio-WestlakeU/audiossl#10]
Hi, I notice that your work is the first one to conduct evaluation on audioset_eval_strong dataset, this is really amazing. Recently, I am going to evaluate my method on this set too, so there are some details I want to ask you,
1. Does the 'w/o var-pen' mean that the parameter 'alpha_st' in the psds function equals to 0?
2. While computing the psds2 score on the set, I find that the process gets stuck due to the high CPU demanding, have you ever met the same problem?
3. Since no weak or unlabeled data is available, is the model trained solely with frame-level binary cross-entropy loss on the 100000+ training set?
I am looking forward to your reply, many thanks.

**Comment by SaoYear** [comment]
Hi,

Appreciate a lot for your interest! 

The experiments on the **Strongly labeled AudioSet** is meaningful for evaluating the SED capacity of models under real applications. But using it is indeed troublesome. Of course, we met the problems you mentioned in our development stage.

I would answer your questions one by one:
1. Yes, the "w/o var-pen" in our table means the $\alpha_{\text{st}} = 0$.
2. We also found the computation of PSDS1 and PSDS2 are very slow, due to the high CPU demanding. We accelerate the process by parallelizing the computation process. You could refer to our updating codes to see the implementation (still need some days...). We reduce the entire inference time to around 10 - 30mins, which is acceptable for us.
3. You are right. We only use a strongly-labeled audio clips + BCE loss to train the model.

Hope these answers make sense to you! If you have any further questions, please let us know.

Nian

**Comment by SaoYear** [comment]
And also, please notice that we calculated PSDS scores by the `psds_eval` package (used in the DCASE 2022) instead of the `sed_scores_eval` package (used in the DCASE 2023). The latter provides a smoothen curve and gives higher scores.

**Comment by Ming-er** [comment]
Thank you for your advice and I will give it a shot. : )
